src/cdk/tree/tree-redesign.spec.ts

```diff
@@ -0,0 +1,1742 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {ComponentFixture, TestBed, fakeAsync, flush} from '@angular/core/testing';
+import {
+  Component,
+  ErrorHandler,
+  ViewChild,
+  TrackByFunction,
+  Type,
+  EventEmitter,
+  ViewChildren,
+  QueryList,
+} from '@angular/core';
+
+import {CollectionViewer, DataSource} from '@angular/cdk/collections';
+import {Directionality, Direction} from '@angular/cdk/bidi';
+import {combineLatest, BehaviorSubject, Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+import {CdkTreeModule, CdkTreeNodePadding} from './index';
+import {CdkTree, CdkTreeNode} from './tree';
+import {getTreeControlFunctionsMissingError} from './tree-errors';
+
+describe('CdkTree redesign', () => {
```

Comment:
Could you say which tests have been added/removed/modified in this fork?

Reply:
done.